src/views/register/content/RegisterCard.vue

```diff
@@ -54,7 +66,9 @@ export default class RegisterCard extends Vue {
   public sendForm (): void {
     this.$refs.registerForm.validate((valid) => {
       if (valid) {
-        console.log(this.registerForm)
+        axios.post('/AuthenticateUser/register', this.registerForm).then(
+          () => this.$router.push('/login')
+        )
```

Comment:
Authentication is not properly implemented on the backend yet, so this section will be reworked when the backend is ready.